src/widgets/faq/ui/faq.tsx

```diff
@@ -1,23 +1,27 @@
 /* eslint-disable @stylistic/jsx-one-expression-per-line */
 import classNames from 'classnames/bind';
 
+import { FaqData } from '../types';
 import { TextWithLink } from '@/shared/ui/text-with-link';
 import { WidgetTitle } from '@/shared/ui/widget-title';
-import { faqData } from 'data';
 
 import styles from './faq.module.scss';
 
 const cx = classNames.bind(styles);
 
-export const Faq = () => {
+type FaqProps = {
+  faqData: FaqData;
+};
+
+export const Faq = ({ faqData }: FaqProps) => {
   return (
-    <section className={cx('faq', 'container')}>
+    <section className={cx('faq', 'container')} data-testid="faq">
       <div className={cx('content', 'info-wrapper')}>
         <WidgetTitle mods="asterisk">FAQ</WidgetTitle>
-        <ol className={cx('list')}>
+        <ol className={cx('list')} data-testid="faq-list">
           {faqData.map(({ question, answer }, index) => (
             <li className={cx('list-item')} key={question}>
-              <span className={cx('question', 'marked')}>
+              <span className={cx('question', 'marked')} data-testid="faq-question">
                 {index + 1}. {question}
               </span>
```

Comment:
```suggestion
              <Subtitle className={cx('question', 'marked')} fontSize="extra-small" data-testid="faq-question">
                  {`${index + 1}. ${question}`}
              </Subtitle>
```
And remove comment on the 1 line, please

Reply:
Done, thanks!